src/plugins/index_pattern_field_editor/public/components/field_editor/field_editor.tsx

```diff
@@ -228,7 +241,21 @@ const FieldEditorComponent = ({
         formFieldPath="__meta__.isFormatVisible"
         withDividerRule
       >
-        <FormatField />
+        {/** TODO types, onChange, onError */}
+        {/** is type es or kbn? */}
+        <FormatField
+          fieldAttrs={{
+            name,
+            type: 'string' as KBN_FIELD_TYPES,
+            esTypes: ['keyword'] as ES_FIELD_TYPES[],
+          }}
+          indexPattern={indexPattern}
+          fieldFormatEditors={fieldFormatEditors}
+          fieldFormats={fieldFormats}
+          uiSettings={uiSettings}
+          onChange={() => {}}
```

Comment:
You probably don't need the `onChange` or `onError` as you handle the "format" form field directly inside the `<FormatField />` component.